db/mysql/migrations/20190909161140_add_cert_type_column.sql

```diff
@@ -0,0 +1,11 @@
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE hostkeys
+ADD COLUMN cert_type ENUM('host_cert', 'user_cert') NOT NULL;
```

Comment:
double-check the behavior of enum, not null, without setting the default value.

Reply:
This might require attention since mysql will just assign a value if it is not specified.

Reply:
From [MySQL docs](https://dev.mysql.com/doc/refman/5.6/en/data-type-defaults.html):
> For ENUM, the default is the first enumeration value.

but lack of DEFAULT value will cause an error if the INSERT skips `cert_type` column.